path_config

```diff
@@ -16,6 +16,11 @@ path /status StatusHandler {
     root .;
 }
 
+path /reverse_proxy ReverseProxyHandler {
+    remote_host localhost;
+    remote_port 1234;
```

Comment:
Oh, okay. I understand I guess. A real world proxy would want to have specifying a port be an option I think, but if that's what they said in discussion than we'll just do it that way.